modules/lvms-monitoring-logical-volume-manager-operator.adoc

```diff
@@ -6,42 +6,56 @@
 [id="lvms-monitoring-using-lvms_{context}"]
 = Monitoring {lvms}
 
-When you use {rh-rhacm} to install {lvms}, you must configure {rh-rhacm} Observability to monitor all the clusters from one place.
+To enable cluster monitoring, you must add the following label in the namespace where you have installed {lvms}:
+[source,text]
+----
+openshift.io/cluster-monitoring=true
```

Comment:
I'm not fully sure on the context of where to apply this command. Add to a config file, assign to an environment variable, or launch a tool and specify the command there?

Examples from the Adding custom metrics link:

![Screenshot from 2024-03-08 10-24-48](https://github.com/openshift/openshift-docs/assets/57954076/0fa2dae9-8fd6-4062-98a4-b239e8d59752)
![Screenshot from 2024-03-08 10-24-51](https://github.com/openshift/openshift-docs/assets/57954076/c8a813fb-6153-4226-b220-d46541e8e9c9)

Reply:
This label must be added in a namespace:

![Screenshot from 2024-03-08 16-43-01](https://github.com/openshift/openshift-docs/assets/148952239/1cbcfa96-3a3c-4330-a560-463966321630)

Click *Edit* to add a new label.

Reply:
Ah, OK. Thanks. This makes sense.